Gemfile.lock

```diff
@@ -29,6 +29,7 @@ GEM
 PLATFORMS
   arm64-darwin-20
   arm64-darwin-21
+  arm64-darwin-23
```

Comment:
Huh? Why did you need to add this?

Reply:
When I uninstalled and reinstalled ruby my Gemfiles changes. I just wanted to keep it consistent with what was on my machine.